src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,42 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
```

Comment:
Set.of(
```suggestion
        numbers = Set.of(1,1,2,3);
```

위와 같이 축약할 수 있을 것 같네요~

Reply:
정확히는 Java 9에서 추가된 메소드입니다!